manager/assets/modext/widgets/modx.panel.welcome.js

```diff
@@ -93,7 +93,7 @@ Ext.extend(MODx.panel.Welcome, MODx.Panel, {
                             }
                             var datestamps = Ext.select(".date_stamp", container);
                             datestamps.each(function (el) {
-                                el.dom.innerText = new Date(el.dom.innerText).format(MODx.config.manager_date_format);
+                                el.dom.innerText = new Date(el.dom.innerText).toLocaleString().format(MODx.config.manager_date_format);
```

Comment:
`toLocaleString` formats the date/time already with a locale. A subsequent formatting with `format` makes no sense. Since it is better to use the `manager_date_format` system setting, I would remove the toLocaleString here. Maybe `manager_time_format` system setting can be used in combination.

Reply:
You're right, it's a bit weird formatting it twice but here's the different outputs I get:

```Date(el.dom.innerText).format(MODx.config.manager_date_format);```

2021-05-29

```new Date(el.dom.innerText).toLocaleString();```

29/05/2021, 3:11:00 am

```new Date(el.dom.innerText).toLocaleString().format(MODx.config.manager_date_format);```

29 May 2021, 3:11 am

Reply:
Please use this, if you need the time too:
```
new Date(el.dom.innerText).format(MODx.config.manager_date_format + ' ' + MODx.config.manager_time_format);
```

The locale in the browser is maybe not the locale set for ExtJS and the Browser shows different languages for the timestamp then i.e. here a german date (browser locale) in an english backend (MODX system setting).

Reply:
Oh I see what you mean, that's a fair point! I'll make that change now. 👍🏻